src/components/FormInput/index.tsx

```diff
@@ -107,3 +112,56 @@ export const FormInput = forwardRef<TextInput, Props>(
     </View>
   ),
 );
+
+interface FormInputDateProps extends Omit<Props, 'onChange' | 'value'> {
+  onChange?: (date: Date) => void;
+  value?: Date;
+}
+
+export const FormInputDate = ({
+  onChange,
+  value = new Date(),
+  ...props
+}: FormInputDateProps) => {
+  const [show, setShow] = useState<boolean>(false);
+  return (
+    <View>
+      <Pressable
+        onPress={() => {
+          setShow(true);
+        }}
+      >
+        <FormInput
+          {...props}
+          pointerEvents="none"
+          editable={false}
+          value={value.toLocaleDateString()}
+        />
+      </Pressable>
+      {show && (
+        <>
+          <DateTimePicker
+            maximumDate={new Date()}
+            testID="dateTimePicker"
+            value={value}
+            mode="date"
+            display={Platform.OS === 'android' ? 'default' : 'spinner'}
+            is24Hour
+            onChange={(event, selectedDate) => {
+              if (!onChange) return;
+              if (Platform.OS === 'android') {
+                setShow(false);
+              }
+              onChange(selectedDate || new Date());
```

Comment:
🍌 이렇게 되면 `onChange` prop이 들어오지 않으면 밑의 `setShow(false)`도 실행되지 않는다는 건데, 이게 의도가 맞을까요? 만약 그게 아니고 `onChange`의 존재 여부만 판단하고 싶으신 거라면 이렇게 하는 게 낫지 않을까요?
```suggestion

              if (Platform.OS === 'android') {
                setShow(false);
              }
              onChange?.(selectedDate || new Date());
```